src/sass/components/topics_cards.scss

```diff
@@ -0,0 +1,42 @@
+h2.topics__header {
+	margin-top: 2rem;
+	margin-bottom: -2rem;
+	@include AU-media( md ) {
+		margin-top: 5rem;
+		margin-bottom: 0;
+	}
+}
+
+.topicscards {
+	padding-top: 3rem;
+}
+
+.topicscards__intro {
+	margin-bottom: 2rem;
+	position: relative;
+	top: 1.5rem;
+}
+
+.topicscards__tagline {
+	margin-top: 1.5rem;
+	@include AU-media( md ) {
+		max-width: 90%;
+	}
+}
+
+h2.topicscards__headline {
+	max-width: 90%;
+}
+
+.topicscards__viewall {
```

Comment:
You should start using the `AU-space` function which uses unit and has pixel fallbacks for older browsers.